templates/docs/upgrade-guide-v3.md

```diff
@@ -81,3 +81,7 @@ We removed the `p-table--sortable` that was previously required to enable sortin
 ### Inline images
 
 We deprecated the inline images component. Please use the logo section component instead.
+
+### Buttons
+
+Neutral button style provided by `p-button--neutral` is exactly the same as default `p-button` styling, so neutral variant is deprecated and will be removed in future version 3.0 of Vanilla. Please use `p-button` instead.
```

Comment:
If they're in the upgrade guide, you could argue they're already in the future (i.e. dealing with the 3.0 upgrade)
```suggestion
Neutral button style provided by `p-button--neutral` was deprecated as it is exactly the same as default `p-button` styling. Please use `p-button` instead.
```

Reply:
We are using similar wording in other places where we mention deprecated class names, so I'll leave that.